src/main/java/com/denizenscript/depenizen/bukkit/events/mythicmobs/MythicMobsDespawnEvent.java

```diff
@@ -0,0 +1,65 @@
+package com.denizenscript.depenizen.bukkit.events.mythicmobs;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.depenizen.bukkit.objects.mythicmobs.MythicMobsMobTag;
+import io.lumine.mythic.bukkit.events.MythicMobDespawnEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+
+public class MythicMobsDespawnEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // mythicmob <'mob'> despawns
+    //
+    // @Location true
+    //
+    // @Triggers when a MythicMob despawns.
+    //
+    // @Context
+    // <context.mob> Returns the MythicMob that is despawning.
+    // <context.entity> Returns the EntityTag for the MythicMob.
+    //
+    // @Plugin Depenizen, MythicMobs
+    //
+    // @Group Depenizen
+    //
+    // -->
+
+    public MythicMobsDespawnEvent() {
+        registerCouldMatcher("mythicmob <'mob'> despawns");
+    }
+
+    public static MythicMobsDespawnEvent instance;
```

Comment:
This field is never used/set, can remove it (events used to all track their instance, but they only do that when actually necessary now)

Reply:
Removed.